index.html

```diff
@@ -86,5 +86,12 @@ <h2 class="tech-name">Django</h2>
       </p>
     </div>
   </div>
+  <div class="footer">
+    <h2 class="footer-title">Contact us!</h2>
+    <div class="footer-container">
+      <input class="footer-input" type="text" name="info">
+      <input class="footer-submit"type="submit" value="SEND">
```

Comment:
there are two ways of doing a submit button, the one you used and this one:
```
<button type="sumbit">
  ...
</button>
```
We tend to use the one I just showed you because yo can set whatever content you want while in the other option it's only text and you have to do it in the attribute instead of inside of it.